Lib9c.Policy/Policy/MaxTransactionsPerSignerPerBlockPolicy.cs

```diff
@@ -18,27 +18,16 @@ private MaxTransactionsPerSignerPerBlockPolicy(int defaultValue)
             new MaxTransactionsPerSignerPerBlockPolicy(int.MaxValue);
 
         public static IVariableSubPolicy<int> Odin =>
-            Default
-                // Note: Introduced to prevent transactions spamming that may result in
-                // the chain grinding to a halt without meaningful state transitions happening.
-                // See https://github.com/planetarium/libplanet/issues/1449.
-                // Issued for v100086.
-                // FIXME: Starting index and value must be finalized accordingly before deployment.
-                .Add(new SpannedSubPolicy<int>(
-                    startIndex: 2_800_001,
-                    value: 4));
+            Default;
```

Comment:
Is it okay to remove this restriction? I'm not sure how `MaxTransactionsPerSignerPerBlockPolicy` affects but many users doesn't have enough 'MEAD's now and their transactions will fail.